docs/en/setup/service-agent/php-agent/README.md

```diff
@@ -55,27 +55,31 @@ make install
 ## Configure php.ini
 
 ```ini
-[skywalking]
+[skywalking_agent]
```

Comment:
So, we change from `skywalking` to `skywalking_agent` now?

Reply:
This tag isn't useful for php, but ours extension name is `skywalking_agent`, so `skywalking_agent` is more accurate.